docs/references/rdctl-command-reference.md

```diff
@@ -0,0 +1,98 @@
+---
+title: rdctl Command Reference
+---
+
+`rdctl` is a command-line tool, included in Rancher Desktop that enables command-line access to the application features. `rdctl` is developed to help users with tasks such as scripting (for automation, CI/CD), troubleshooting, remote management etc. The current version of `rdctl` supports below commands, and support for more commands would be added in the upcoming releases.
+
+***Note:  Rancher Desktop app should be running on your machine to use `rdctl` commands ***
```

Comment:
app must be running

-- It won't work otherwise.

Reply:
Modified per your suggestion.